Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Install dependencies only when needed
-FROM node:16-alpine AS deps
+FROM timbru31/node-alpine-git:16 AS deps
```

Comment:
Why a non officiel image? What are the benefits of this new one?

Reply:
To be honest, I needed node alpine with git and this is the first result from Google and it works 😅
I will look if I can do it with an official node image.

Reply:
Thanks for pointing it out, I think the fix is pretty easy. I test it and push if it's working.

Reply:
Fixed

Reply:
Thanks a lot for your justification and time. 👍🏻 

![bitmoji](https://sdk.bitmoji.com/render/panel/efb4c0d2-74ee-4867-9beb-1c89f6800757-a1987ddb-a11e-49bc-8a9b-052f30b8f6ce-v1.png?transparent=1&palette=1&width=246)